bedrock/products/urls.py

```diff
@@ -10,7 +10,6 @@
 urlpatterns = (
     path("vpn/", views.vpn_landing_page, name="products.vpn.landing"),
     path("vpn/invite/", views.vpn_invite_page, name="products.vpn.invite"),
-    path("vpn/invite/waitlist/", views.vpn_invite_waitlist, name="products.vpn.invite.waitlist"),
```

Comment:
Note for reviewer: this URL will now 404 since posting to basket is now handled client side.